Terraform/emty_folde_for_terraform.txt

```diff
@@ -0,0 +1 @@
+this is empty folde for terraform
```

Comment:
an alternative solution could be: 
Create an empty file called .gitkeep in the directory, and git add it.

Reply:
folder or file?

Reply:
file

.gitkeep is a common thing you will see in codebases, where an empty folder needs to be tracked via Git.

The name of the dummy file may not always be .gitkeep but you'll see the actual practice over and over again as a developer.